specs/agents/mobile/session.md

```diff
@@ -0,0 +1,19 @@
+# Session 
+
+### Overview
+A `session` is a collection of `logs`, `events`, `transactions` and `spans` (`LETS`) associated with a specific device within a specific period of time. 
```

Comment:
(A very minor thing) shouldn't we mention metrics as well? or is it covered by one of these items already mentioned?

Reply:
since metrics are aggregated they wont have any relation to sessions

Reply:
I see, thanks for the clarification